kin-core/src/main/java/kin/core/PaymentWatcher.java

```diff
@@ -61,6 +61,34 @@ public void onEvent(TransactionResponse transactionResponse) {
             });
     }
 
+    /**
+     * Start watching for create account, use {@link #stop()} to stop watching, startCreateAccountListener can be called again after stop.
+     * <p><b>Note:</b> Events will be fired on background thread.</p>
+     *
+     * @param listener listener object for payment events
+     */
+    public void startCreateAccountListener(final WatcherListener<PaymentInfo> listener){
```

Comment:
I think CreateAccountWatcher should return a different event object, something like `AccountCreationInfo` with the created account and initial balance (the similarity between account creation and payment not necessarily clear to the sdk client)

Reply:
I think that sdk user will not understand create account info - basically it similar to payment as it involve receiving native assets as the account created.

Reply:
it's clear to us that it's similar, but it's not trivial that creating Kin account involved other account funding an account with initial 'native' asset.

Reply:
Stellar considers CREATE_ACCOUNT to be a payment, which is why it shows up in the payments stream.  For this requirement, we don't actually care _which_ payment we observe.  Observing _any_ payment means the account has been created.

Reply:
my point exactly- the create account is specific to stellar. on Ethereum/bitcoin for example there is no special "create account" operation the client side create keypair and the blockchain is aware of the account only when is has balance (he can only have balance if some other address send ether to this newly created public address)